SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,16 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||adtg.widerplanet.com^
+||adx.turl.co.kr^
+||ssp.realclick.co.kr^
+||kitweb.tadapi.info^
+||mediacategory.com^$third-party
+||adimg.wisenut.co.kr^
```

Comment:
ads are loaded from this domain?
I don't see all these requests on wisenut.co.kr

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/a2f01aa318abfc99817e47ae2cbe80c7193bfee9